src/flash.rs

```diff
@@ -34,3 +307,94 @@ impl ACR {
         unsafe { &(*FLASH::ptr()).acr }
     }
 }
+
+/// Opaque AR register
+pub struct AR {
+    _0: (),
+}
+
+#[allow(dead_code)]
```

Comment:
Why are these allow(dead_code) needed? Is rustc missing the usages, or are they currently unused?

Reply:
Those are currently unused; I added them for completeness.